### PR TITLE
reorg

### PR DESCRIPTION
**Pull request by StarfallProjects**
This PR:
- Adds to the README and CONTRIBUTING docs
- Reorganizes the documentation to take advantage of tabs
- Updates all links within the docs
- Includes some proofreading work, particularly starting to standardize headings and UI element references [wip]
- Adds a brief introduction to CodeSee Local

### CODE REVIEW

**Comment by pchocho** [comment]
Hi, I like the reorganization into tabs. We did intentionally remove the TOC and set a max-width (feedback from Julia that the right column made the page UX confusing), but I think I'm fine with how this is looking now. It's just odd when there isn't much content on a page, or when the headers don't feel like TOC headers.

I had added an &nbsp; at the end of the content of each page to add extra margin to the bottom. It feels too close to the bottom nav now. If we could do this via CSS instead that would be great.

**Comment by StarfallProjects** [comment]
> I had added a non-breaking space at the end of the content of each page to add extra margin to the bottom. It feels too close to the bottom nav now. If we could do this via CSS instead that would be great.

Yup 100% - I removed it because expecting folks to remember every time they create a page is always risky. Should be easy to do with CSS. Will do that as part of this PR.

File: mkdocs.yml [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
-    - Next: setup-next.md
-    - Nuxt: setup-nuxt.md
-    - Storybook: setup-storybook.md
-    - Gatsby: setup-gatsby.md
-    - Ember: setup-ember.md
-    - Generic setup: setup-generic.md
-    - Configuration options: options.md
+  - 👋 Welcome:
+    - Introduction: index.md
+    - 🙏 Troubleshooting: troubleshooting.md
```
Comment (ramonh) [inline mkdocs.yml:5]: Shall we remove this emoji to keep the subpages' not having them consistent?

**Review by ramonh — APPROVED** [review]
Looks great @StarfallProjects thank you so much!

Left more of an open-ended question, but lgtm!

File: mkdocs.yml (hunk 1 above)
Comment (StarfallProjects) [reply mkdocs.yml]: Sounds good. Removed.

**Comment by ramonh** [comment]
Nice one, thank you so much!

**Pull request merged**